src/server/zset_family.cc

```diff
@@ -2599,6 +2636,296 @@ void ZSetFamily::GeoDist(CmdArgList args, ConnectionContext* cntx) {
                              distance_multiplier);
 }
 
+// Search all eight neighbors + self geohash box
+int membersOfAllNeighbors(ConnectionContext* cntx, string_view key, const GeoHashRadius* n,
+                          GeoShape* shape, GeoArray& ga, unsigned long limit) {
```

Comment:
We don't take mutable references to arguments in Dragonfly, only const-ref or pointer. Please convert `ga` to be of type `GeoArray*`

Reply:
And could you take `shape` by const-ref, i.e. `const GeoShape& shape`?
(you'll need to use `&shape` below when you pass it to `geoWithinShape()`)